.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,55 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[Feature request]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        For general questions and design discussion, please use the Discussions page: https://github.com/open-rmf/rmf/discussions
+        Not sure if this is the right repository? Open an issue on https://github.com/open-rmf/rmf
+        For bug reports, please use the bug report issue form.
+        Be as detailed about your issue as possible.
+
+  - type: checkboxes
+    id: checked-issues
+    attributes:
+      label: Before proceeding, is there an existing issue or discussion for this?
+      options:
+        - label: I have done a search for similar [issues](.?q=type%3Aissue) and [discussions](https://github.com/open-rmf/rmf/discussions).
+          required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Describe in a few sentences what the feature consists of and what problem it will solve.
+        Include why you think the feature is necessary, who else will benefit, and why they will benefit.
+        e.g. "When I am ... I want to be able to ... so that I can ..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: implementation-considerations
+    attributes:
+      label: Description
```

Comment:
Noticed that there's a repeated label `Description` here, is this intended?

Reply:
Oops, good catch, it should have been `Implementation Considerations`, https://github.com/open-rmf/.github/pull/4/commits/58745f14d2d14c416a5145151a21ed520911c740